cases/index.html

```diff
@@ -1,6 +1,7 @@
 ---
 layout: default
 title: 'Cases ____ deranged'
+sitemap-include: 'yes'

```

Comment:
I'd prefer underscores in variable names (e.g. `sitemap_include`) or camelCase (e.g. `sitemapInclude`)  to avoid confusion with the minus operator `-` which in *some* languages (but not Liqiuid or YAML!) menas something different.

Reply:
Fixed